vendor/github.com/xenserver/go-xenserver-client/vm.go

```diff
@@ -521,6 +521,21 @@ func (self *VM) SetIsATemplate(is_a_template bool) (err error) {
 	return
 }
 
+func (self *VM) GetOtherConfig() (other_config map[string]string, err error) {
```

Comment:
@makunterry - this looks good, but can we please open a pull request upstream in the go-xenserver-client so the commit in this repo is just updating the vendored dependency? (given we own both repos!).